convert2rhel/toolopts.py

```diff
@@ -200,8 +202,9 @@ def _register_options(self):
         group.add_option(
             "-c",
             "--config-file",
-            help="Configuration file containing"
-            " password or activation key for subscription-manager."
+            help="A configuration file to safely provide either a user password or an activation key for registering"
+            " the system through subscription-manager. Alternatively, passing these values through the"
+            " --activationkey or --password option would leak them through a list of running processes."
```

Comment:
Just wanted to open this comment for a question, wouldn't be better to use “would leak sensitive information through a list of running processes” instead of specifying that the activation key or password would leak?

Reply:
I feel that we can cover everything with “sensitive information” in this context, even if we find other types of things that can be leaked, as this kind of text is way general everywhere.

Reply:
Currently the config file is used just for the two specific pieces of information.
I expect that once we add more config options we'll need to update the description here.
Till then IMHO it's better to be specific - this way it's clear. The way you propose it would make it open to questions and assumptions.